make/images.mk

```diff
@@ -461,6 +461,11 @@ ubuntu18:
 ubuntu18-azure:
```

Comment:
I think you're going to need to change this to `build-gcp-ubuntu-1804` 

What I did previously is map this one to the new target see here https://github.com/mesosphere/konvoy-image-builder/commit/bdbd8592e7f43b77c32960c95c607d12247d8f62#diff-1d0eb908228c37e62c17adb04039870d318703e16c43c002a9e6de2c981e5774

Reply:
great catch @faiq